lib/media/segment_reference.js

```diff
@@ -450,6 +465,44 @@ shaka.media.SegmentReference = class {
     return this.independent;
   }
 
+  /**
+   * Mark the reference as partial.
+   *
+   * @export
+   */
+  markAsPartial() {
+    this.partial = true;
+  }
+
+  /**
+   * Returns true if the segment is partial.
+   *
+   * @return {boolean}
+   * @export
+   */
+  isPartial() {
+    return this.partial;
+  }
+
+  /**
+   * Mark the reference as last partial.
```

Comment:
This function (and the next one) should explain what "last partial" is more clearly.
Something along the lines of "Mark the reference as being the last part of the full segment."

Reply:
Done!